src/tigerbeetle/main.zig

```diff
@@ -97,7 +101,7 @@ const Command = struct {
         os.close(command.dir_fd);
     }
 
-    pub fn format(allocator: mem.Allocator, cluster: u32, replica: u8, path: [:0]const u8) !void {
+    pub fn format(allocator: mem.Allocator, options: SuperBlock.FormatOptions, path: [:0]const u8) !void {
```

Comment:
Debating whether I need to move `options` to be the last argument in the chain of format calls. I think it makes sense to do that, but I'd rather do it in a follow up.